examples/integration/test/fixtures/bwb.xcodeproj/xcshareddata/xcschemes/iOSApp.xcscheme

```diff
@@ -78,9 +78,9 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "CC4C6C2A28BA30B1838E405C"
+               BlueprintIdentifier = "64AE6000A6317B9C077F3B1E"
                BuildableName = "CoreUtilsObjC.framework"
-               BlueprintName = "CoreUtils"
+               BlueprintName = "CoreUtilsObjC"
```

Comment:
Example 2: `CoreUtilsObjC.framework` is the product, so it makes sense that this target is `CoreUtilsObjC` instead of `CoreUtils`.